### ISSUE TITLE
rustdoc: the required const feature gate is not shown

### ISSUE THREAD

**Issue by RalfJung** [rust-lang/rust#131618]
For regular unstable functions, rustdoc shows something like this

![Image](https://github.com/user-attachments/assets/6bed7f26-2633-43e2-93c7-42e8a5919708)

However, for const-unstable functions, all we get is

![Image](https://github.com/user-attachments/assets/d20e6faa-e585-4481-9c50-f097444962b5)

There's no indication of what the feature gate is. Worse, if the function is unstable and const-unstable, but under different feature gates, then there's not even the "const: unstable" that links to the issue -- there's no way to figure out what the const feature gate is other than looking at the source.

Cc @rust-lang/rustdoc

**Comment by RalfJung** [comment]
Turns out the "const: unstable" has a tooltip indicating the const feature gate. But that can't be used for copy-paste, and it is quite hard to notice.

**Comment by RalfJung** [comment]
> The biggest problem I see is how to handle this on mobile devices.

Long-press could show the tooltip.

Also, if we can fix this for desktop users only, that's still better than the status quo. Most of us don't program on a mobile device, I assume, and so having the docs work well on large formfactors is quite relevant.

**Comment by lolbinarycat** [comment]
yeah, i was thinking we could just tweak the current rendering to `(const: unstable, feature_flag_name)`, and then simply hide the feature flag name on mobile devices using css.

**Comment by GuillaumeGomez** [comment]
Shouldn't be a tooltip but full-fledged text thing like feature-gate items.

**Comment by GuillaumeGomez** [comment]
Anyway, opened https://github.com/rust-lang/rust/pull/135830.
